### DISCUSSION TITLE
Make my own using modules

### DISCUSSION THREAD

**Discussion by denisjoshua** [tetele/onju-voice-satellite#58] [General]
Hi there, I have a stupid question... 
Because on PcbWay I must buy 5 pieces, and I don't need it for now, 
and the cost is too much for me... 
...can I buy the:
- ESP32­-S3 (8M PSRAM) 
- 2 X INMP441 mic. 
- MAX98357 amp 
- 3 Watt 8 Ohm Speaker, 

...put all on a Bareboard,
and use your code to make it work please ? 

Thanks in advance 
(mersi mult :-)
Denis

**Comment by tetele** [comment]
Of course. There are tons of configs which are ready made for that specific combo. This one is tailored for the Onju Voice with all its bells and whistles, like touch controls  LEDs etc. but you can really trim it down if you put your mind to it.

I recommend this config as a better starting point https://github.com/BigBobbas/esphome_firmware/tree/main/ESPHome%20Diy%20Voice%20Builds

**Reply by denisjoshua** [reply]
Greate :-) 
Thanks a lot for the confirmation and the link... 
Denis
